test/jdk/java/awt/font/JNICheck/FreeTypeScalerJNICheck.java

```diff
@@ -45,7 +45,7 @@ public static void main(String[] args) throws Exception {
         } else {
             ProcessBuilder pb = ProcessTools.createTestJavaProcessBuilder("-Xcheck:jni", FreeTypeScalerJNICheck.class.getName(), "runtest");
             OutputAnalyzer oa = ProcessTools.executeProcess(pb);
-            oa.shouldContain("Done").shouldNotContain("WARNING").shouldHaveExitValue(0);
+            oa.shouldContain("Done").shouldNotContain("WARNING").shouldNotContain("AWT Assertion").shouldHaveExitValue(0);
```

Comment:
Maybe wrap each condition to its own line?
```suggestion
            oa.shouldContain("Done")
              .shouldNotContain("WARNING")
              .shouldNotContain("AWT Assertion")
              .shouldHaveExitValue(0);
```
This way each verified condition stands out.

Reply:
Done.